app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,23 @@
 class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    user = User.find(params[:user_id])
+    post = Post.find(params[:post_id])
```

Comment:
- [ ] On this method you are doing the same as I explained above, Please make sure that you create a comment that belongs to the current_user and the posts you are in.

Reply:
I have implemented all required changes, thanks